acrg/BC/timevarying_BC.py

```diff
@@ -32,15 +32,23 @@
 -------
 
 '''
-if 'os'    not in dir(): import os
-if 'np'    not in dir(): import numpy as np
-if 'xr'    not in dir(): import xarray as xr
-if 'paths' not in dir(): from acrg_config.paths import paths
+import os
+import json
+import numpy as np
+import xarray as xr
+from scipy import interpolate
+from collections import OrderedDict
+
+from acrg.name.name import open_ds
+from acrg.config.paths import Paths
+from acrg.convert import concentration
+from acrg.countrymask import domain_volume
+
     
 class BoundaryConditions:
     def __init__(self, vmr_var, gph_height_var, dataset=None, filename=None,
-                 file_dir=None, time_coord='time', height_coord='height', species=None,
```

Comment:
Was there a reason the `height_coord` input was removed here? Is this no longer used?

Reply:
It looks like it was the same as gph_height_var, I've taken out the reamining refs to it